packages/cursorless-engine/src/languages/markdown.ts

```diff
@@ -60,15 +55,12 @@ type HeadingMarkerType = typeof HEADING_MARKER_TYPES[number];
  * @returns A node finder that will return the next node that is of the same
  * marker level or higher than the original type
  */
-function makeMinimumHeadingLevelFinder(
-  headingType: HeadingMarkerType,
-): NodeFinder {
+function makeMinimumHeadingLevelFinder(headingType: HeadingMarkerType): NodeFinder {
   const markerIndex = HEADING_MARKER_TYPES.indexOf(headingType);
   return (node: SyntaxNode) => {
     return node.type === "atx_heading" &&
-      HEADING_MARKER_TYPES.indexOf(
-        node.firstNamedChild?.type as HeadingMarkerType,
-      ) <= markerIndex
+      HEADING_MARKER_TYPES.indexOf(node.firstNamedChild?.type as HeadingMarkerType) <=
+        markerIndex
```

Comment:
This seems to be less readable, at least whilst prettier still wraps lines after binary operators.